src/Common/src/System/Globalization/IdnMapping.cs

```diff
@@ -98,71 +83,36 @@ public String GetAscii(String unicode, int index, int count)
             // We're only using part of the string
             unicode = unicode.Substring(index, count);
 
-            return GetAsciiUsingOS(unicode);
-        }
-
-        [System.Security.SecuritySafeCritical]
-        private String GetAsciiUsingOS(String unicode)
-        {
             if (unicode.Length == 0)
```

Comment:
I wasn't sure the best thing to do with these checks.  For example, ideally I'd think this would be "if (count == 0) throw new ArgumentOutOfRangeException("count", ...);" in the up-front contract section, but that'd be a change from the existing behavior.  Thoughts?